.github/workflows/ci.yml

```diff
@@ -98,6 +98,9 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
```

Comment:
Is this required? Capping python in pyproject.toml should be sufficient was my expectation.

Reply:
It doesn't seem to be respecting it because the generated project is not using uv yet. It's for the `test_bare.sh` script

Reply:
I'll removing it to confirm

Reply:
The pytest run uses Python 3.10 here `platform linux -- Python 3.10.12`:

https://github.com/cookiecutter/cookiecutter-django/actions/runs/11306934810/job/31448084312?pr=5456#step:7:561

This is because these lines use the first `pip` and `pytest` on the PATH, which is not the one from uv:

https://github.com/cookiecutter/cookiecutter-django/blob/822c1c0327d2025ccde10ec1085f0ecdd7df184a/tests/test_bare.sh#L21

https://github.com/cookiecutter/cookiecutter-django/blob/822c1c0327d2025ccde10ec1085f0ecdd7df184a/tests/test_bare.sh#L24